lib/onebox/engine.rb

```diff
@@ -42,7 +42,7 @@ def options=(opt)
 
     def initialize(url, timeout = nil)
       @errors = {}
-      @options = DEFAULT
+      @options = DEFAULT.dup
```

Comment:
Mutation was happening in https://github.com/discourse/discourse/blob/c161f5e0b298fa6842b83d2bc4627b5a8f92aff0/lib/onebox/engine.rb#L39